packages/react/src/ReactFetch.js

```diff
@@ -74,7 +74,8 @@ if (enableCache && enableFetchInstrumentation) {
         url = resource;
       } else {
         // Normalize the request.
-        const request = new Request(resource, options);
+        // we clone the request as to not disturb the body (in the event it's a ReadableStream)
+        const request = resource.clone();
```

Comment:
The `resource` here can still be a string when options is non-empty. So needs to handle that case.